webpack.config.js

```diff
@@ -398,11 +398,15 @@ module.exports = {
             },
             {
                 test: /\.css$/,
-                loader: extract.extract("css-loader?minimize=&root=" + libdir)
```

Comment:
@martinpitt I don't understand what the 'root=" + libdir' was used for; can you give me a heads up?

Reply:
This was introduced in commit 904ef8662288e26328f5f6f49d5a3e1b7aff3cea three years ago. I'm afraid I can't interpret this either, but if it builds and works, then your change is fine :-) Can you please force-push and trigger tests?